packages/run-protocol/test/amm/constantProduct/test-checkInvariants.js

```diff
@@ -2,15 +2,24 @@
 // eslint-disable-next-line import/no-extraneous-dependencies
 import { test } from '@agoric/zoe/tools/prepare-test-env-ava.js';
 import { AmountMath } from '@agoric/ertp';
-import { BASIS_POINTS } from '../../../../src/contracts/constantProduct/defaults.js';
+import { makeRatio } from '@agoric/zoe/src/contractSupport/index.js';
+import { BASIS_POINTS } from '../../../src/vpool-xyk-amm/constantProduct/defaults.js';
 import { setupMintKits } from './setupMints.js';
-import { makeRatio } from '../../../../src/contractSupport/index.js';
 import {
   pricesForStatedInput,
   pricesForStatedOutput,
-} from '../../../../src/contracts/constantProduct/calcSwapPrices.js';
-import { checkKInvariantSellingX } from '../../../../src/contracts/constantProduct/invariants.js';
-import { getXY } from '../../../../src/contracts/constantProduct/getXY.js';
+} from '../../../src/vpool-xyk-amm/constantProduct/calcSwapPrices.js';
+import { checkKInvariantSellingX } from '../../../src/vpool-xyk-amm/constantProduct/invariants.js';
+import { getXY } from '../../../src/vpool-xyk-amm/constantProduct/getXY.js';
+
+/**
+ * @typedef {Object} SwapPriceArgs
+ * @property {Amount} amountGiven
+ * @property {PoolAllocation} poolAllocation
+ * @property {Amount=} amountWanted
+ * @property {Ratio} protocolFeeRatio
+ * @property {Ratio} poolFeeRatio
+ */
```

Comment:
more (welcome) type tweaks... seems worth a mention in the commit message.

Reply:
added a few lines to the message.